applications/RANSApplication/custom_utilities/rans_check_utilities.cpp

```diff
@@ -21,12 +21,13 @@ namespace Kratos
 {
 namespace RansCheckUtilities
 {
-bool CheckIfModelPartExists(const Model& rModel, const std::string& rModelPartName)
+bool CheckIfModelPartExists(
```

Comment:
Can't the `HasModelPart` of the `Model`  be used to remove this?

Reply:
done :)